dolphinscheduler-worker/src/main/java/org/apache/dolphinscheduler/server/worker/processor/TaskExecuteResultAckProcessor.java

```diff
@@ -54,7 +54,7 @@ public void process(Channel channel, Command command) {
             logger.error("task execute response ack command is null");
             return;
         }
-        logger.info("task execute response ack command : {}", taskExecuteAckMessage);
+        logger.info("Receive task execute response ack command : {}", taskExecuteAckMessage);
```

Comment:
Please move this log after `LoggerUtils.setTaskInstanceIdMDC(taskExecuteAckMessage.getTaskInstanceId());`
This will add traceId in log header.